_redirects

```diff
@@ -1,5 +1,6 @@
 /about-us/our-values /about-us/values
 /how-we-work/client-guide /client-guide/onboarding
+/mentoring-people/running-one-on-ones /mentoring-people/running-1-1
```

Comment:
In this case (1:1s), for the URL, I prefer using words. So, numbers for titles and words for URLs.

Reply:
Done! ✅